lib/schmobile.rb

```diff
@@ -1,5 +1,11 @@
 require 'rack'
 
+begin
+  require 'action_dispatch'
```

Comment:
This ensures that `ActionDispatch::Request` is loaded before we apply the request patch.